docs/testing/debug-add-ins-using-f12-tools-ie.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Debug add-ins using developer tools for Internet Explorer
+description: 'Debug add-ins using the developer tools in Internet Explorer'
+ms.date: 10/20/2021
+ms.localizationpriority: medium
+---
+
+# Debug add-ins using developer tools in Internet Explorer
```

Comment:
Do these instructions apply to certain Office applications/hosts?

Reply:
No.